docs/appendices/release-notes/5.8.5.rst

```diff
@@ -47,6 +47,12 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue which would allow to cast a number to a
+  :ref:`NUMERIC<type-numeric>`, even if the ``NUMERIC``'s' precision and scale
+  would not be sufficient to fit the number, without loosing precision, e.g.::
+
+      SELECT CAST(123.45 as NUMERIC(4, 2))
```

Comment:
Shouldn't we also add that now we throw an error?

Reply:
I thought it's implied, since the issue is that we `would allow`, but can do.

Reply:
Not insisting)

Reply:
https://github.com/crate/crate/pull/16857/commits/dc1982f7a7dd24045443286a8d47ef869b7e5ca2

always better to have clear message, thx!